src/PrestaShopBundle/Resources/config/services/adapter/cart.yml

```
@@ -128,3 +128,15 @@ services:
     tags:
       - name: tactician.handler
         command: 'PrestaShop\PrestaShop\Core\Domain\Cart\Command\AddProductToCartCommand'
+
+  prestashop.adapter.cart.command_handler.delete_cart_handler:
+    class: 'PrestaShop\PrestaShop\Adapter\Cart\CommandHandler\DeleteCartHandler'
```

Not so long ago we started autowiring (but long enough that I must require this change, sorry :smile: ). Could you go through your yml definitons and autowire everything (and use public: false where possilbe).
```suggestion
  PrestaShop\PrestaShop\Adapter\Cart\CommandHandler\DeleteCartHandler:
    autowire: true
    public: false
```